lrauv_ignition_plugins/src/ScienceSensorsSystem.cc

```diff
@@ -761,14 +758,18 @@ bool ScienceSensorsSystemPrivate::comparePclPoints(
 /////////////////////////////////////////////////
 void ScienceSensorsSystemPrivate::FindTrilinearInterpolators(
   pcl::PointXYZ &_pt,
-  std::vector<int> &_inds,
-  std::vector<float> &_sqrDists,
   std::vector<int> &_interpolatorInds1,
   std::vector<pcl::PointXYZ> &_interpolators1,
   std::vector<int> &_interpolatorInds2,
   std::vector<pcl::PointXYZ> &_interpolators2,
   int _k)
 {
+  // TODO(tfoote) magic numbers
+  // Should be passed in and paramaterized based on the expected data 
+  // distribution height or calculated from the grandularity of the dataset.
+  float slice_depth = 5; // 5 meter. 
```

Comment:
Hmm, I see how this would simplify the search, but I think it makes the algorithm less robust. As you said, 5 meters is a magic number. It is very specific to the one specific data set we have. Right now, we don't have a way of calculating the resolution, and we know that resolution will depend on where you are in the data. If we upstream with this specific resolution, it's not going to work in the general case. The current code works in a more general case, whatever the granularity the data. For that, I think the complexity is worth it. We're not running into performance issues to need to simplify it.